app/views/new_admin/allocation_chart/_allocations.html.erb

```diff
@@ -0,0 +1,76 @@
+<table class="w-full whitespace-no-wrap ">
+  <thead>
+  <tr class="text-xs font-semibold tracking-wide text-left text-gray-500 uppercase border-b dark:border-gray-700 bg-gray-50 dark:text-gray-400 dark:bg-gray-800">
+    <th class="px-4 py-3"><%= I18n.t('name') %></th>
+    <th class="px-4 py-3"><%= I18n.t('client') %></th>
+    <th class="px-4 py-3"><%= I18n.t('allocated_until') %></th>
+    <th class="px-4 py-3"><%= I18n.t('level') %></th>
+    <th class="px-4 py-3"><%= I18n.t('specialty') %></th>
+    <th class="px-4 py-3"><%= I18n.t('english_evaluation') %></th>
+    <th class="px-4 py-3"><%= I18n.t('skills') %></th>
+  </tr>
+  </thead>
+  <tbody class="bg-white divide-y dark:divide-gray-700 dark:bg-gray-800">
+  <% allocations.each do |allocation| %>
+    <tr class="text-gray-700 dark:text-gray-400">
+      <td class="px-4 py-3">
+        <div class="flex items-center text-sm gap-1">
+          <%= gravatar_for allocation[:user], { size: 30, css_class: 'object-cover rounded-full' } %>
+          <p class="text-sm text-gray-600 dark:text-gray-400 font-semibold">
+            <%= allocation[:name] %>
+          </p>
+        </div>
+        </div>
+      </td>
+      <td class="px-4 py-3">
+        <div class="flex items-center text-sm">
+          <p class="text-sm text-gray-600 dark:text-gray-400">
+            <%= allocation[:project_name] %>
+          </p>
+        </div>
+        </div>
+      </td>
+      <td class="px-4 py-3">
+        <div class="flex items-center text-sm">
+          <p class="text-sm text-gray-600 dark:text-gray-400">
+            <%= allocation[:allocation_end_at] %>
+          </p>
+        </div>
+        </div>
+      </td>
+      <td class="px-4 py-3">
+        <div class="flex items-center text-sm">
+          <p class="text-sm text-gray-600 dark:text-gray-400">
+            <%= allocation[:level].capitalize if allocation[:level] %>
```

Comment:
doesn't the decorator handle empty values already? Also, if `level` is set as `'N/A'`, `capitalize` will turn into `'N/a`

https://github.com/Codeminer42/Punchclock/blob/0dba9a9bf4011fbb236769d42ec5f18ce573e446/app/decorators/user_decorator.rb#L10-L12